iosApp/Survey/Sources/Presentation/Modules/ResetPassword/ResetPasswordView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ResetPasswordView.swift
+//  Survey
+//
+//  Created by Bliss on 26/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResetPasswordView: View {
+
+    @State private var email: String = ""
+    @State private var loading = false
+
+    var body: some View {
+        EmbedLoadingView(loading: loading) {
+            ZStack {
+                GeometryReader { geometry in
+                    Assets.backgroundBlur
+                        .image
+                        .resizable()
+                        .aspectRatio(contentMode: .fill)
+                        .ignoresSafeArea()
+                        .frame(maxWidth: .infinity, maxHeight: .infinity)
+
+                    VStack(
+                        alignment: .center,
+                        spacing: 20.0
+                    ) {
+                        Spacer().frame(maxHeight: geometry.size.height / 10.0)
+
+                        Assets.logoWhite.image
+
+                        Text(Localize.resetPasswordTextInstruction())
+                            .multilineTextAlignment(.center)
+
+                        Spacer().frame(maxHeight: 70.0)
+
+                        emailField
+                        resetButton
+                    }
+                    .padding(.horizontal, 24.0)
```

Comment:
The more of these padding/spacing values will be used in the application the more screens we have. So I suggest you to create constants values for these for later reusing and modification such as `smallPadding`, `mediumPadding`, `largePadding`, etc. What do you think?

Reply:
Added in https://github.com/blyscuit/kmm-swiftUI-IC/pull/56/commits/8edbbc2a81db456ef46a1258d58b109b6a24d252